packages/react/src/codemods/emotion-to-compiled/__tests__/emotion-to-compiled.test.tsx

```diff
@@ -336,7 +362,7 @@ describe('emotion-to-compiled transformer', () => {
 
     const Component = () => (
       <ClassNames>
-        {({ css, cx }) => (
+        {({ css }) => (
           <SomeComponent
             wrapperClassName={css({ color: 'green' })}
             css={c\`background-color: green \`}
```

Comment:
we need to remove `c` from here

Reply:
😕 `css as c` is coming from import. And its the input. Its removed from output. I am confused now 😅

Reply:
oh, im a dumb dumb 😄